src/ucs/debug/memtrack.c

```diff
@@ -402,5 +400,3 @@ int ucs_memtrack_is_enabled()
 {
     return ucs_memtrack_context.enabled;
 }
-
-#endif
```

Comment:
seems the separation to ucs_memtrack_do_allocated was not ported from master , any reason?

Reply:
artefact of conflict resolving 
fixed